.github/workflows/build_release_apk.yml

```diff
@@ -69,5 +71,5 @@ jobs:
           TELEGRAM_CI_BOT_TOKEN: ${{ secrets.TELEGRAM_CI_BOT_TOKEN }}
           TELEGRAM_CI_CHANNEL_ID: ${{ secrets.TELEGRAM_CI_CHANNEL_ID }}
           TELEGRAM_TEAM_GROUP_ID: ${{ secrets.TELEGRAM_TEAM_GROUP_ID }}
-          ARTIFACT_DIRECTORY: artifacts/release-apk
+          ARTIFACT_DIRECTORY: artifacts/release-apk/dark/release
```

Comment:
Why it's `artifacts/release-apk/dark/release`?

Reply:
I have added this to ship dark(default) version alone to telegram channel. Let mw know If it's required to upload other varient to telegram channel as well. I will check and update it.

Reply:
You should upload all flavors?